### PR TITLE
i18n(es): update `auto.mdx` & `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by Mardroide**
#### Description
- Update content

#### Reference
- [b6f0f0c](https://github.com/withastro/docs/commit/b6f0f0c491e4b94635c473660dbb0830b44fa549)
- [configuration-reference.mdx](https://github.com/withastro/docs/commits/main/src/content/docs/en/reference/configuration-reference.mdx?since=2023-11-14T16:09:57.000Z)

### CODE REVIEW

File: src/content/docs/es/install/auto.mdx [hunk 1]
```diff
@@ -84,6 +84,8 @@ Cada plantilla de inicio posee un script preconfigurado que ejecutará `astro de
 
 Astro escuchará cualquier cambio en la carpeta `src/` y actualizará automáticamente tu proyecto. De esta forma, no será necesario reiniciar el servidor local durante el desarrollo.
 
+Cuando visualices tu sitio en el navegador, tendrás acceso a la barra de herramientas de desarrollo de Astro (/es/reference/configuration-reference/#dev-toolbar-options). A medida que construyas, te ayudará a inspeccionar tus [islas](/es/concepts/islands/), detectar problemas de accesibilidad y mucho más.
```
Comment (Waxer59) [inline src/content/docs/es/install/auto.mdx:87]:
```suggestion
Cuando visualices tu sitio en el navegador, tendrás acceso a la [barra de herramientas de desarrollo de Astro](/es/reference/configuration-reference/#dev-toolbar-options). A medida que construyas, te ayudará a inspeccionar tus [islas](/es/concepts/islands/), detectar problemas de accesibilidad y mucho más.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by sarah11918** [comment]
Hi! I can see the problem here is that the `/es/` configuration-reference page has not yet been updated to include the new Dev Toolbar Options!

You have two choices... 1) update configuration-reference FIRST  :smile: OR 2) remove the link here now, then come back and put the link back in after configuration reference is updated!

**Comment by Mardroide** [comment]
Hi, finally I just updated the `configuration-reference.mdx` page. Also changed the `/es/` link. Thanks! 🐳

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -757,33 +723,20 @@ Puedes usar comodines para definir los valores `hostname` y `pathname` permitido
   - Terminar con '/**' para permitir todas las sub-rutas ('startsWith').
   - Terminar con '/*' para permitir solo un nivel de sub-ruta.
 
-## Opciones de Markdown
+## Opciones de la barra de herramientas de Astro Dev
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:726]:
```suggestion
## Opciones de la barra de herramientas de desarrollo
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 3]
```diff
@@ -757,33 +723,20 @@ Puedes usar comodines para definir los valores `hostname` y `pathname` permitido
...
 **Tipo:** `boolean`<br />
-**Por defecto:** `false`
+**Por defecto:** `true`
 </p>
 
-Controla si las páginas Markdown de borrador deben incluirse en la compilación.
+Si deseas activar la barra de herramientas de Astro Dev. Esta barra de herramientas te permite inspeccionar las islas de tu página, ver auditorías útiles sobre rendimiento y accesibilidad, y mucho más.
 
-Una página de Markdown se considera un borrador si incluye `draft: true` en el frontmatter. Las páginas de borrador siempre se incluyen y son visibles durante el desarrollo (`astro dev`), pero de forma predeterminada no se incluirán en la compilación final.
+Esta opción se aplica a todo el proyecto, para desactivar la barra de herramientas sólo para ti, ejecuta `npm run astro preferences disable devToolbar`. Para desactivar la barra de herramientas para todos tus proyectos Astro, ejecuta `npm run astro preferences disable devToolbar --global`.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:737]:
```suggestion
Esta opción se aplica a todo el proyecto, para desactivar la barra de herramientas sólo para ti, ejecuta `npm run astro preferences disable devToolbar`. Para desactivar la barra de herramientas para todos tus proyectos de Astro, ejecuta `npm run astro preferences disable devToolbar --global`.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 4]
```diff
@@ -1019,20 +933,22 @@ El idioma predeterminado de tu sitio/aplicación. Este es un campo obligatorio.
...
 </p>
 
-Una lista de todos los idiomas admitidos por el sitio web (por ejemplo, `['en', 'es', 'pt-br']`). Esta lista también debe incluir el `defaultLocale`. Este es un campo obligatorio.
+Una lista de todos los idiomas admitidos por el sitio web, incluyendo el `defaultLocale`. Este es un campo obligatorio.
+
+Los idiomas pueden enumerarse como códigos individuales (por ejemplo, `['en', 'es', 'pt-br']`) o asignarse a una `ruta` compartida de códigos (por ejemplo, `{ path: "english", code: ["en", "en-US"]}`). Estos códigos se utilizarán para determinar la estructura de URL de tu sitio desplegado.
 
-Ningún formato o sintaxis de idioma en particular está obligado, pero la estructura de tu carpeta debe coincidir exactamente con los idiomas de la lista.
 
+No se impone ningún formato o sintaxis de código de idioma en particular, pero las carpetas de tu proyecto que contengan tus archivos de contenido deben coincidir exactamente con los elementos `locales` de la lista. En caso de que varios "códigos" apunten a un prefijo de ruta URL personalizado, almacena los archivos de contenido en una carpeta con el mismo nombre que la "ruta" configurada.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:949]:
```suggestion
No se impone ningún formato o sintaxis de código de idioma en particular, pero las carpetas de tu proyecto que contengan tus archivos de contenido deben coincidir exactamente con los elementos `locales` de la lista. En caso de que varios `códigos` apunten a un prefijo de ruta URL personalizado, almacena los archivos de contenido en una carpeta con el mismo nombre que la `ruta` configurada.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 5]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
 
-**Tipo:** `'prefix-always' | 'prefix-other-locales'`<br />
-**Por defecto:** 'prefix-other-locales'<br />
-<Since v="3.5.0" />
+**Tipo:** `Routing`<br />
+<Since v="3.7.0" />
 </p>
 
-Controla la estrategia de enrutamiento para determinar las URL de tu sitio. Configura esto en función de la configuración de ruta de carpeta/URL para tu idioma predeterminado:
+Controla la estrategia de enrutamiento para determinar las URLs de tu sitio. Establézcalo en función de la configuración de la ruta de la carpeta/URL para tu idioma predeterminado.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:990]:
```suggestion
Controla la estrategia de enrutamiento para determinar las URLs de tu sitio. Establécelo en función de la configuración de la ruta de la carpeta/URL para tu idioma predeterminado.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 6]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+
+## Flags legacy
 
-   Las URLs serán de la forma `example.com/[lang]/content/` para todas las rutas, incluido el idioma predeterminado.
-   Directorios localizados se utilizan para todos los idiomas, incluido el predeterminado.
+Para ayudar a algunos usuarios a migrar entre versiones de Astro, ocasionalmente introducimos flags `legacy`.
+Estas flags te permiten optar por algunos comportamientos desactualizados u obsoletos de Astro
+en la última versión, para que pueda continuar actualizándose y aprovechar los nuevos lanzamientos de Astro.
+
+## Flags Experimentales
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1015]:
```suggestion
## Banderas Experimentales
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 7]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+
+Cuando es `false`, sólo los idiomas no predeterminados mostrarán un prefijo de idioma.
+El `defaultLocale` no mostrará un prefijo de idioma y los archivos de contenido no existen en una carpeta localizada.
+ Las URLs tendrán la forma `example.com/[lang]/content/` para todos los idiomas no predeterminados, pero `example.com/content/` para la configuración regional predeterminada.
+
+Cuando es `true`, todas las URLs mostrarán un prefijo de idioma.
+Las URLs serán de la forma `example.com/[lang]/content/` para cada ruta, incluyendo el idioma por defecto.
+Se utilizan carpetas localizadas para cada idioma, incluido el predeterminado.
+
+## Flags legacy
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1009]:
```suggestion
## Banderas legacy
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 8]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
 
-   Las URLs serán de la forma `example.com/[lang]/content/` para todas las rutas, incluido el idioma predeterminado.
-   Directorios localizados se utilizan para todos los idiomas, incluido el predeterminado.
+Para ayudar a algunos usuarios a migrar entre versiones de Astro, ocasionalmente introducimos flags `legacy`.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1011]:
```suggestion
Para ayudar a algunos usuarios a migrar entre versiones de Astro, ocasionalmente introducimos banderas `legacy`.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 9]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+Estas flags te permiten optar por algunos comportamientos desactualizados u obsoletos de Astro
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1012]:
```suggestion
Estas banderas te permiten optar por algunos comportamientos desactualizados u obsoletos de Astro
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 10]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+en la última versión, para que pueda continuar actualizándose y aprovechar los nuevos lanzamientos de Astro.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1013]:
```suggestion
en la última versión, para que puedas continuar actualizando y aprovechar los nuevos lanzamientos de Astro.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 11]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+
+## Flags Experimentales
+
+Astro ofrece flags experimentales para dar a los usuarios acceso temprano a nuevas características.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1017]:
```suggestion
Astro ofrece banderas experimentales para dar a los usuarios acceso temprano a nuevas características.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 12]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+Estas flags no son garantía de ser estables.
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1018]:
```suggestion
No se garantiza que estas banderas sean estables.
```

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 13]
```diff
@@ -1049,38 +965,79 @@ Usa este objeto para declarar una ruta de `locale` de reserva para cada idioma q
...
+
+**Tipo:** `boolean`<br />
+**Por defecto:** `false`<br />
+<Since v="2.10.4" />
+</p>
+
+Evita que los scripts de los componentes no utilizados se incluyan en una página de forma inesperada.
+La optimización se realiza al máximo esfuerzo y puede que no incluya los scripts utilizados. Asegúrate de volver a comprobar las páginas creadas
+antes de publicar.
+Habilita la optimización del análisis de scripts elevados añadiendo la flag experimental:
```
Comment (Waxer59) [inline src/content/docs/es/reference/configuration-reference.mdx:1032]:
```suggestion
Habilita la optimización del análisis de scripts elevados añadiendo la bandera experimental:
```

File: src/content/docs/es/install/auto.mdx [hunk 14]
```diff
@@ -84,6 +84,8 @@ Cada plantilla de inicio posee un script preconfigurado que ejecutará `astro de
 
 Astro escuchará cualquier cambio en la carpeta `src/` y actualizará automáticamente tu proyecto. De esta forma, no será necesario reiniciar el servidor local durante el desarrollo.
 
+Cuando visualices tu sitio en el navegador, tendrás acceso a la [barra de herramientas de desarrollo de Astro](/es/reference/configuration-reference/#barra-de-herramientas-de-astro-dev). A medida que construyas, te ayudará a inspeccionar tus [islas](/es/concepts/islands/), detectar problemas de accesibilidad y mucho más.
```
Comment (Waxer59) [inline src/content/docs/es/install/auto.mdx:87]:
```suggestion
Cuando visualices tu sitio en el navegador, tendrás acceso a la [barra de herramientas de desarrollo de Astro](/es/reference/configuration-reference/#opciones-de-la-barra-de-herramientas-de-desarrollo). A medida que construyas, te ayudará a inspeccionar tus [islas](/es/concepts/islands/), detectar problemas de accesibilidad y mucho más.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by Mardroide** [comment]
Thanks! All done! 🙌

**Pull request merged**